src/PageNotFound.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default () => {
```

Comment:
let's add name for this component and adhere to a uniform naming convention for components